decentraland/unity-renderer
# feat: create channels window

## What does this PR change?

Adds the posibility to create channels through a window.

## How to test the changes?

1. Open the conversation list
2. Click on the `+` button
3. It should open a channel creation window
4. Write nearby, it should throw an error
5. Write a channel that already exists, it should throw an error
6. Check that the max limit of the name of the channel is `17`
7. Clear the channel name text, the button should be disabled
8. Click the create button
9. The channel window should be opened
10. Check that the channel has been added in the conversation list
11. Go to the channel browser and click the create button on the top of the window, repeat process
12. Go to the channel browser, search for any channel. There should be a create option displayed at the bottom of the list. Click it.
13. Modify the `ChatChannelsControllerMock.GetFakeChannels` method to return 0 channels. Go to the channel browser and check that a new option to create a channel is added. Click it.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
